test/test_harness.js

```diff
@@ -0,0 +1,80 @@
+var log4js = require('log4js');
+var mongoose = require('mongoose');
+var path = require('path');
+var util = require('util');
+
+var mongodbFs = require('../lib/mongodb-fs');
+
+function TestHarness(mocks, port, logLevel) {
+  this.config = {
+    port: port || 27027,
+    mocks: mocks,
+    log: {level: logLevel || process.env.LOG_LEVEL || 'warn'}
+  };
+  this.logger = this.createLogger(__filename);
```

Comment:
No, `TestHarness` does not inherit from anything. Previously, the tests were only using loggers to report stages of set-up or tear-down, but now `TestHarness` does that. Should a test need a logger explicitly, it can always grab one from `TestHarness`:
```js
var logger = TestHarness.createLogger(__pathname);
logger.info("I'm here!");
```